### PR TITLE
Add the new Effort section to the DG and fix some bugs

### PR DESCRIPTION
**Pull request by UncleGrandpa925**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1721,10 +1722,52 @@ The team has put in a tremendous amount of effort to this project, with a single
...
+### Compared to AB3
+Our app can be thought of as a superset of AB3. AB3 was concerned primarily with the management of contacts. We recognised that contacts and friends were a crucial part of any student’s life. Hence, we decided to build upon AB3 by altering the app to be suited for students, and so that the app could help students conveniently manage their student lives.
+
+Hence, we decided to implement module, lesson, and task managers on top of the existing AB3 contacts manager. Now, students will be able to keep track of all their academic and social commitments with a single app, where they previously had to use 4. 
+
+We put in a lot of effort to add these 3 extra features, working late most nights before our v1.2 demo. Furthermore, we decided after that we should integrate all 4 core features seamlessly, to improve upon the existing user experience. As such, for v1.3 users were able to:
+* view all tasks, lessons, and contacts related to a module
+* view all upcoming tasks, and lessons
+* view all future tasks
+Again, this was no easy feat, and our team worked tirelessly to finish our features before v1.3.
```
Comment (simonteozw) [inline docs/DeveloperGuide.md]: I think the first letter of the bullet points should be capitalised

**Review by simonteozw — COMMENTED** [review]
Looks good! Just 1 comment

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (UncleGrandpa925) [reply docs/DeveloperGuide.md]: Okay sure

**Review by simonteozw — APPROVED** [review]
LGTM

**Pull request merged**